pyproject.toml

```diff
@@ -19,6 +19,11 @@ pytest = "^6.2.4"
 pytest-cov = "^2.12.1"
 pylint = "^2.8.3"
 black = "^21.5b2"
+isort = "^5.8.0"
+ipython = "^7.24.1"
+
```

Comment:
Can you add rich here? It's causing the build to fail.

Reply:
I added a requirements file generated from poetry with `poetry export -f requirements.txt --output requirements.txt` and have CI set to use that, hopefully that fixes the issue